nextflow.config.example

```diff
@@ -32,8 +32,9 @@ params {
     dir = "${PWD}/output"
     sample_dir = { "${params.output.dir}/${sample_id}" }
     publish_mode = "link"
-    publish_downloaded_fastq = true
-    publish_trimmed_fastq = true
+    publish_sra = false
```

Comment:
Do we want thee `publish_sra`, `publish_fastq` and `publish_trimmed_fastq` to be false by default?  I would think the most common use case would be a small dataset and in that case folks probably want all of this data....